mysql/tests/test_statements.py

```diff
@@ -853,7 +853,11 @@ def test_async_job_cancel(aggregator, dd_run_check, dbm_instance):
 
 
 def _expected_dbm_instance_tags(dbm_instance):
-    return dbm_instance.get('tags', []) + ['server:{}'.format(common.HOST), 'port:{}'.format(common.PORT)]
+    return dbm_instance.get('tags', []) + [
+        'server:{}'.format(common.HOST),
+        'port:{}'.format(common.PORT),
+        'dbms_flavor:{}'.format(MYSQL_FLAVOR.lower()),
```

Comment:
Is it better to use f"" formatting? it's a little bit less verbose.

Reply:
`f""` is more concise. i use the same `.format` syntax for consistency. we used `.format` in the past because `f""` is python3 only. it's no longer a concern now.